src/probnum/randprocs/markov/continuous/_sde.py

```diff
@@ -7,14 +7,13 @@
 import scipy.linalg
 
 from probnum import randvars
+from probnum.randprocs.markov import _transition, discrete
+from probnum.randprocs.markov.continuous import _utils
```

Comment:
Don't mean to be petty, but does that contradict our contribution guidelines? And if yes, do we still want to keep it in order to save letters?

Reply:
And on a related note (and connecting to a similar comment below), it might be dangerous to import `_utils` sometimes from `continuous` and other times from `discrete`. Maybe we should live with the long `x.y.z.<...>` lines?

Reply:
Ah nevermind, this is just to avoid relative imports... Please ignore

Reply:
this is to avoid cyclic imports. However, I see your point about it being a bit confusing.

I think this problem could be resolved by never using the filename `_utils.py`, but instead `_sde_utils.py`, `_discrete_utils.py`, etc.?